jobs/package-dockertested/Jenkinsfile

```diff
@@ -92,21 +92,106 @@ node('buildvm-devops') {
 			stage ('Build an Origin release') {
 				sh """{
 cat <<EOF
-sudo su origin
 set -o errexit -o nounset -o pipefail -o xtrace
 cd /data/src/github.com/openshift/origin
 hack/build-base-images.sh
-make release
+OS_BUILD_ENV_PRESERVE=_output/local hack/env OS_ONLY_BUILD_PLATFORMS='linux/amd64' hack/build-rpm-release.sh
+sudo systemctl restart docker
```

Comment:
Funny story. `docker cp` takes RAM proportional to what you copy, which in the `hack/env` case often is > a couple GB. The daemon also holds onto that RAM for about ten minutes after the `docker cp` client call is finished. This was killing our jobs, so we just restart the daemon because a dead process can't sit on memory ;)